packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/step.st

```diff
@@ -0,0 +1,8 @@
+simulation
+step
+	"Step only if the simulation is running."
+	
+	self isSimulationRunning
+		ifTrue: [
+			super step.
```

Comment:
Call it or not? 
If we will make the decision in xcom, you could already approve the merge request, don't you?
@Pottiman @pitwegner